public/assets/styles/modules/container.scss

```diff
@@ -5,8 +5,12 @@
 
   text-align: left;
 
+  @include media(medium) { width: 94%; }
```

Comment:
Why 94%? That seems random. Can this be a function of the padding like `calc(100% - 3em);` ?